HdmiInput/doc/HdmiInputPlugin.md

```diff
@@ -333,18 +361,26 @@ Returns the current EDID value.
 ```json
 {
     "jsonrpc": "2.0",
-    "id": 1234567890,
+    "id": 42,
     "result": {
-        "EDID": "",
+        "EDID": "...",
         "success": true
     }
 }
 ```
 
 <a name="method.startHdmiInput"></a>
-## *startHdmiInput <sup>method</sup>*
+## *startHdmiInput [<sup>method</sup>](#head.Methods)*
 
 Activates the specified HDMI Input port as the primary video source.
+ 
+### Events 
+| Event | Description | 
+| :----------- | :----------- | 
+| `onInputStatusChanged` | Triggers the event when activates HDMI Input source and Input status changes to `started` | 
```

Comment:
Change "activates HDMI Input source" to "HDMI Input source is activated".

Reply:
Updated the changes

Reply:
Updated the changes